src/components/DropDownMenu/DropDownMenu.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { memo } from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  visiblePeople: Person[];
+  onSelect: (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    person: Person,
+    status: boolean
+  ) => void;
+};
+
+export const DropDownMenu:React.FC<Props> = memo(({
+  visiblePeople,
+  onSelect,
+}) => {
+  const onSelected = (
```

Comment:
The 'onSelected' function is duplicating the functionality of the 'onSelect' prop. You could pass the 'onSelect' function directly to the onClick handler, provided the event signatures match.